requirements.txt

```diff
@@ -38,5 +38,4 @@ requests==2.18.4
 simplegeneric==0.8.1
 six==1.11.0
 traitlets==4.3.2
-urllib3==1.22
```

Comment:
Discard what I said, the urllib3 module is a requirement for requests so it needs to stay here. Sorry !